charts/routes/README.md

```diff
@@ -79,7 +79,7 @@ The following table lists the configurable parameters of the Routes chart and th
 | global.mlp.uiServiceName | string | `"mlp"` |  |
 | global.mlp.useServiceFqdn | bool | `false` |  |
 | global.mlp.vsPrefix | string | `"/api"` |  |
-| global.oauthclient | string | `"test-client-123"` |  |
+| global.oAuthClient | string | `"test-client-123"` |  |
```

Comment:
nit: raised in the other MR as well, can we change this to `oAuthClientId`?

Reply:
done